.github/workflows/scheduled_test.yaml

```diff
@@ -0,0 +1,41 @@
+name: Scheduled Test
+
+on:
+  schedule:
+    # Execute weekly on Monday UTC time. Offset time to avoid high load period on github servers.
+    - cron: "15 3 * * 1"
+
+jobs:
+  integration-test:
+    name: Integration tests (lxd)
+    uses: ./.github/workflows/integration_test.yaml
+  notify-on-failure:
+    name: Notify on failure
+    needs: [integration-test]
+    if: ${{ inputs.notify && failure() }}
+    runs-on: ubuntu-latest
+    env:
+      USERNAME: ${{ secrets.email_username }}
+      PASSWORD: ${{ secrets.email_password }}
+    steps:
+      - name: Check credential exists
+        if: ${{ env.USERNAME == null || env.PASSWORD == null }}
+        run: |
+          echo "Missing email username and password as input for this workflow."
+          exit 1
+      - name: Notify Failure
+        uses: dawidd6/action-send-mail@v3
+        with:
+          server_address: smtp.gmail.com
+          server_port: 465
```

Comment:
I think the `server_address` and `server_port` should be put in the secret as well so we can update them without modifying the code.

Reply:
Done.